spec/features/request_ride_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "rails_helper"
+
+feature "Request ride" do
+  let(:user) { create(:user) }
+  let!(:ride) { create(:ride) }
+  before(:each) { sign_in user }
+
+  context "when ride available" do
+    before(:each) do
+      visit rides_path
+      click_on "Take ride"
+      expect{click_on "Confirm"}.to change{ride.passengers.count}.by(1)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.